src/setup/node.rs

```diff
@@ -85,7 +85,7 @@ impl Node {
                 }
                 Some(exit_code) => Some(format!("crashed with exit code {}", exit_code)),
             };
-
+            child.wait()?;
```

Comment:
Here's what happened: 
Test always passed when run only this one and occasionally failed with 'connection refused' at `synth_node.node().connect(node.addr()).await.unwrap();` when I run all tests quickly multiple times. (So in other words: connect was fine first, then refused later). I figured, that's because some delay before OS kills child process from previous test run, but I'm willing to accept other explanation.